web_model_read_only/tests/test_models.py

```diff
@@ -0,0 +1,82 @@
+from odoo.exceptions import AccessError
+from odoo.tests.common import SavepointCase
+
+
+class TestIrModel(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestIrModel, cls).setUpClass()
+        cls.readonly_group = cls.env.ref("base.group_user")
+        cls.group_manager = cls.env.ref("base.group_erp_manager")
+        cls.ir_model = cls.env["ir.model"].create(
```

Comment:
For what you create new model? Could you use existing model for example `res.partner`?